pkg/client/client.go

```diff
@@ -63,3 +71,63 @@ func (n Node) Join(ctx context.Context, topic string) *Topic {
 
 	return t
 }
+
+func (n Node) Ls(ctx context.Context, path []string) (cluster.AnchorIterator, error) {
+	if len(path) == 0 {
+		it, releae := n.view.Iter(ctx)
+		return cluster.HostAnchorIterator{Vat: n.vat, It: it, Release: releae}, nil
+	}
+
+	a, err := n.hostAnchor(ctx, path)
+	if err != nil {
+		return nil, err
+	}
+
+	return a.Ls(ctx)
+
+}
+
+func (n Node) Walk(ctx context.Context, path []string) (cluster.Anchor, error) {
+	if len(path) == 0 {
+		// TODO: return a RootAnchor that is equivalent to this same implementation
+		return nil, ErrInvalidPath
+	}
+
+	a, err := n.hostAnchor(ctx, path)
+	if err != nil {
+		return nil, err
+	}
+
+	if len(path) > 1 {
+		return a.Walk(ctx, path[1:])
+	} else {
+		return a, nil
+	}
+
+}
+
+func (n Node) hostAnchor(ctx context.Context, path []string) (cluster.Anchor, error) {
+	fut, release := n.view.Lookup(ctx, peer.ID(path[0]))
```

Comment:
`path[0]` will contain the base58-encoded string, but raw `peer.ID`s are random strings containing arbitrary unicode data.  We'll need to use `peer.IDFromString()`.

More generally, we don't need the extra call to `Lookup`, since the vat is capable of resolving hosts from peer ID's alone.  So this whole function can be simplified to:

```go
id, err := peer.IDFromString(path[0])
if err != nil {
    return err
}

return cluster.HostAnchor{
    Peer: id,
    Vat: n.vat,
}, nil
```